src/monaca.js

```diff
@@ -759,6 +759,90 @@
    * @method
    * @memberof Monaca
    * @description
+   *   Checks if the project can be built.
+   * @return {Promise}
+   * @example
+   *   monaca.checkBuildAvailability('myProjectID', 'android', 'debug').then(
+   *     function() {
+   *       //Build the project
+   *     },
+   *     function(err) {
+   *       //Cannot build the project
+   *     }
+   *   );
+   */
+  Monaca.prototype.checkBuildAvailability = function(projectId, platform, buildType) {
+    var deferred = Q.defer();
+
+    if (typeof platform === undefined) {
+      deferred.resolve();
+      return deferred.promise;
+    } else if (typeof buildType === undefined) {
+      deferred.reject(new Error("missing --platform parameter."));
+      return deferred.promise;
+    }
+
+    this._get('/project/' + projectId + '/can_build_app')
+    .then(
+      function(response) {
+        var data;
+
+        try {
+          data = JSON.parse(response);
+        } catch (err) {
+          return deferred.reject(new Error(JSON.stringify(err)));
+        }
+
+        if (data.status === 'ok') {
+          var platformContent = data.result[platform];
+
+          if (!platformContent) {
+            return deferred.reject(new Error("specified platform is not supported."));
+          } else if (platformContent.has_remaining_slot !== true) {
+            return deferred.reject(new Error("you cannot build more apps today."));
```

Comment:
Your plan does not allow further builds at the moment. Please upgrade your account to start build, or try again later.